### PR TITLE
added light and dark theme

### PR DESCRIPTION
**Pull request by Rohanmond**
Added light and dark theme and it will persist inside localstorage

### CODE REVIEW

File: src/components/Nav/Nav.jsx [hunk 1]
```diff
@@ -96,13 +101,32 @@ export const Nav = () => {
...
+              {/* )} */}
+              {/* {token && (
                 <span
                   onClick={() => navigate('/profile')}
                   class='material-icons-outlined nav-link-icon'
                 >
                   person
                 </span>
-              )}
+              )} */}
```
Comment (romabulani) [inline src/components/Nav/Nav.jsx:129]: Remove the comments if not required

File: src/pages/Profile/Profile.css [hunk 2]
```diff
@@ -40,9 +41,10 @@
 .profile-details-item {
   font-size: 1.5rem;
   font-weight: 700;
+  color: var(--text-color);
 }
 .profile-details-label {
-  color: var(--yellow-200);
+  /* color: var(--); */
```
Comment (romabulani) [inline src/pages/Profile/Profile.css:47]: here also remove comment

**Review by romabulani — APPROVED** [review]
- Loved the theme change and UI
- Remove the comments not required
- Good to merge

**Review by hrishi82 — APPROVED** [review]
code looks good! can be merged!

**Review by anuj-kumary — APPROVED** [review]
- Functionality is working as expected
- UI is looking great with themes
- Good to merge

**Pull request merged**